doc/recipe/preprocessor.rst

```diff
@@ -901,24 +941,89 @@ days in a year may vary between calendars, (sub-)daily data are not supported.
 .. code-block:: yaml
 
     preprocessors:
-      multi_model_preprocessor:
+      multi_model_save_input:
```

Comment:
Can you move this section up so it directly follows "...set the parameter `keep_input_datasets` to `false` (default value is `true`).?